client.go

```diff
@@ -3,26 +3,32 @@ package shopify
 import (
 	"os"
 
-	graphqlclient "github.com/r0busta/go-shopify-graphql/v4/graphql"
 	"github.com/r0busta/graphql"
 	log "github.com/sirupsen/logrus"
+	graphqlclient "github.com/xiatechs/go-shopify-graphql/v4/graphql"
 )
 
 const (
-	shopifyAPIVersion = "2021-01"
+	// TODO:
```

Comment:
in an ideal world we would be able to confidently change this, considering the test coverage and that we only need one functionality I would be ok with disabling everything else until it's (if) necessary